static/data/.gitignore

```diff
@@ -1 +1,2 @@
 /lms
+lms_vehicles_registered
```

Comment:
Why do we need to ignore this directory if we commit it?

Reply:
why /lms is ignored if we commit it ?

Reply:
In `lms` there's just sample data committed, but you're supposed to download the rest of the data there too, and we don't want that committed. It doesn't mean it's the right thing to do... And anyway, in `lms_vehicles_registered` you already added everything, didn't you? Or is there more data expected to be put there without being committed?

Reply:
original files are downloaded from lms site in Excel format (because we are working with csv, i did not what to enter new file format lib) - the files are direct save to csv without any change.

I am confuses with the demo files, we save some files in git as a source for init dev data, so currently the exist files are all exist data lms can supply us, I guess next year we will have some new files (2 files at most every year). 

please advice what to leave and where to store the rest of the data ?

Reply:
Generally it's better not to commit data files and instead refer users/developers to download them elsewhere, but if it's just a few kb and there are no licensing issues then I guess it's ok. We'll just commit the new files when we get them next year. In any case it makes no sense to ignore this directory, so please remove it from `.gitignore`.

Reply:
I agree that data files should not be part of the project file system, why we do not have a maintaining storage for them only ? if we have, were is it so i place the files there and leave small amount here..
currently storage is 180Kb for all files ... if you tell me were to store them i will remove them to minimum :)

Reply:
We don't really have a dedicated storage location for data files. We just upload them to Google Driver as a zip file and make them publicly accessible, with the link in our readme file.